### PR TITLE
ref(smartSearchBar): open Read the Docs in new window

### PR DESCRIPTION
**Pull request by olivier-w**
While searching for a key you can now open the docs in a new window. Once you find what you're looking for, you are back in the focused search bar state that you left from, instead of the current flow of going back in browser history and reloading the page you came from.

### Before
https://user-images.githubusercontent.com/465250/187994726-15117bd2-e1c1-496a-9ef0-053e3884e9a6.mp4

### After
https://user-images.githubusercontent.com/465250/187994746-17ec584a-6a86-40a4-a4fa-522db4d42e0d.mp4

### CODE REVIEW

File: static/app/components/smartSearchBar/searchDropdown.tsx [hunk 1]
```diff
@@ -95,7 +95,12 @@ const SearchDropdown = ({
             </Button>
           ))}
       </ButtonBar>
-      <Button size="xs" href="https://docs.sentry.io/product/sentry-basics/search/">
+      <Button
+        size="xs"
+        href="https://docs.sentry.io/product/sentry-basics/search/"
+        target="_blank"
+        rel="noopener noreferrer"
```
Comment (evanpurkhiser) [inline static/app/components/smartSearchBar/searchDropdown.tsx:102]: use `external` instead of these two props

Comment (evanpurkhiser) [reply static/app/components/smartSearchBar/searchDropdown.tsx:102]: I think there is a `openInNewTab` prop too

Comment (evanpurkhiser) [reply static/app/components/smartSearchBar/searchDropdown.tsx:102]: https://github.com/getsentry/sentry/blob/b2839cf8352c17913298c69ce640497b192a11b2/static/app/components/links/externalLink.tsx#L8 actually we may have to add that prop into `Button`

Comment (olivier-w) [reply static/app/components/smartSearchBar/searchDropdown.tsx]: it seems like `externalLink` [defaults `openInNewTab` as true](https://github.com/getsentry/sentry/blob/4ebf875314334710ba2696d42c7d5a0bc203bfe0/static/app/components/links/externalLink.tsx#L12), so we don't need to add that prop to open this button in a new window.

**Review by vuluongj20 — APPROVED** [review]

**Pull request merged**